src/Umbraco.Web/Mvc/RenderModelBinder.cs

```diff
@@ -66,7 +67,7 @@ public override object BindModel(ControllerContext controllerContext, ModelBindi
         // to
         // { RenderModel, RenderModel<TContent>, IPublishedContent }
         //
-        public static object BindModel(object source, Type modelType, CultureInfo culture)
+        public object BindModel(object source, Type modelType, CultureInfo culture)

```

Comment:
This is a breaking change

Reply:
Defo. Only reason was to have an instance as "sender" for the error event but can work around it. Fixing.